mod-source-record-manager-server/src/main/java/org/folio/dao/IncomingRecordDaoImpl.java

```diff
@@ -24,14 +26,30 @@ public class IncomingRecordDaoImpl implements IncomingRecordDao {
 
   private static final Logger LOGGER = LogManager.getLogger();
   public static final String INCOMING_RECORDS_TABLE = "incoming_records";
+  private static final String GET_BY_ID_SQL = "SELECT * FROM %s.%s WHERE id = $1";
   private static final String INSERT_SQL = "INSERT INTO %s.%s (id, job_execution_id, incoming_record) VALUES ($1, $2, $3)";
 
   @Autowired
   private PostgresClientFactory pgClientFactory;
 
+  @Override
+  public Future<Optional<IncomingRecordDto>> getIncomingRecordDtoById(String id, String tenantId) {
+    LOGGER.debug("getById:: Get IncomingRecordDto by id = {} from the {} table", id, INCOMING_RECORDS_TABLE);
```

Comment:
looks like you meant  getIncomingRecordDtoById instead of getById in both messages